app/blogs/[host]/[slug]/page.tsx

```
@@ -0,0 +1,143 @@
+                    className="inline-block bg-secondary/50 text-secondary-foreground hover:bg-secondary text-xs px-3 py-1 rounded transition-colors"
+                  >
+                    {tag.name}
+                  </Link>
+                ))}
+              </div>
+            </div>
+            {/* Post Content */}
+            <div className="prose prose-invert max-w-none mb-12">
+              <div
+                className="pose text-foreground dark:text-foreground"
+                dangerouslySetInnerHTML={{ __html: post.content.html }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)